autofill/autofill-impl/src/test/java/com/duckduckgo/autofill/configuration/RealRuntimeConfigurationWriterTest.kt

```diff
@@ -16,40 +16,49 @@
 
 package com.duckduckgo.autofill.configuration
 
+import com.duckduckgo.autofill.jsbridge.response.AvailableInputTypeCredentials
 import com.squareup.moshi.Moshi.Builder
-import org.junit.Assert
+import org.junit.Assert.assertEquals
 import org.junit.Test
 
 class RealRuntimeConfigurationWriterTest {
     private val testee = RealRuntimeConfigurationWriter(Builder().build())
 
     @Test
     fun whenGenerateResponseGetAvailableInputTypesThenReturnAvailableInputTypesJson() {
-        val expectedJson = "{\n" +
-            "  \"credentials\": false,\n" +
-            "  \"email\": true\n" +
-            "}"
-        Assert.assertEquals(
+        val expectedJson = """
```

Comment:
This is functionally different; adopting the new API format:

- `credentials` is no longer a single boolean
- `username` and `password` availability is individually provided now instead